src/Frame.js

```diff
@@ -0,0 +1,37 @@
+var Bowl = function() {
+  this.frameScore = 0;
+  this.maximumFrameScore = 10;
+  this.pins = 10;
+  this.maximumRollScore = this.pins;
+  this.frameNumber = 1;
+  this.maximumFrames = 10;
+  this.strikes = 0;
+  this.spares = 0;
+};
+
+Bowl.prototype.nextFrame = function() {
+  this.frameNumber++;
+  if (this.frameNumber > this.maximumFrames) this.frameNumber = this.maximumFrames;
```

Comment:
Generally, stick to splitting your if statements over multiple lines, with brackets - it makes for clearer reading